sqlmesh/schedulers/airflow/client.py

```diff
@@ -198,6 +198,9 @@ def apply_plan(
         models_to_backfill: t.Optional[t.Set[str]] = None,
         end_bounded: bool = False,
         ensure_finalized_snapshots: bool = False,
+        directly_modified_snapshots: t.List[SnapshotId] = [],
```

Comment:
We should never use `[]` / `{}` as default arguments in Python. See, for example, this https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
oh nice I didn't know about this, thanks